docs/user/attestation-provider.md

```diff
@@ -0,0 +1,113 @@
+# Becoming an Attestation Provider
+
+This guide is a walk through to become an [attestation provider](#attestation-provider).
+There are two situations in which it makes sense to do so:
+
+* Running local attestation providers, without any capital requirement, to act as a safeguard against malicious forks ;
+* Running an incentivized attestation provider to reap rewards, which contributes to the network and helps improve the decentralization of the network and provides attestations for the [state connector](#state-connector).
+
+??? note "Rewards"
```

Comment:
For such a short note, I prefer to use standard non-collapsible annotations. One less click.

Reply:
Wanted to hide it under the carpet but you're right 😁😁😁

Reply:
Fixed in [`699c109` (#14)](https://github.com/flare-foundation/docs/pull/14/commits/699c109c962540b9b188c4a370aba2f4c6f361b4)